### PR TITLE
chore(performance): Remove project threshold flags

### PR DESCRIPTION
**Pull request by shruthilayaj**
Removes the `project-transaction-threshold` and
`project-transaction-threshold-override` flags so
the feature is available to anyone with `performance-view`

Related [Issue](https://github.com/getsentry/sentry/issues/28379)

### CODE REVIEW

File: static/app/views/performance/data.tsx [hunk 1]
```diff
@@ -473,7 +447,7 @@ function generateGenericPerformanceEventView(
 }
 
 function generateBackendPerformanceEventView(
-  organization: LightWeightOrganization,
+  _organization: LightWeightOrganization,
```
Comment (shruthilayaj) [inline static/app/views/performance/data.tsx:450]: Will clean this up in a follow up PR.

File: src/sentry/conf/server.py [hunk 2]
```diff
@@ -976,8 +976,6 @@ def create_partitioned_queues(name):
     "organizations:dashboards-basic": True,
     # Enable custom editable dashboards
     "organizations:dashboards-edit": True,
-    # Enable dashboards manager.
-    "organizations:dashboards-manage": False,
```
Comment (Zylphrex) [inline src/sentry/conf/server.py:980]: Was this intentional?

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -1058,10 +1056,6 @@ def create_partitioned_queues(name):
     "organizations:release-archives": False,
     # Enable the new release details experience
     "organizations:release-comparison": False,
-    # Enable the project level transaction thresholds
-    "organizations:project-transaction-threshold": False,
-    # Enable the transaction level thresholds
-    "organizations:project-transaction-threshold-override": False,
```
Comment (Zylphrex) [inline src/sentry/conf/server.py:1064]: Might be a good idea to delete the flags themselves in a follow up after all references to them have been removed so we don't experience any hiccups in some intermediate state.

**Review by Zylphrex — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 2 above)
Comment (shruthilayaj) [reply src/sentry/conf/server.py:980]: yes 😛  references to this flag were removed months ago

File: src/sentry/conf/server.py (hunk 3 above)
Comment (shruthilayaj) [reply src/sentry/conf/server.py:1064]: good idea

**Review by dashed — APPROVED** [review]

**Pull request merged**